docs/apidoc.md

```diff
@@ -143,3 +137,20 @@ Example
 In order for the endpoint to save data in the current session
 - the session name must start with the microservice name Example: @apiSuccess (Session) {String} microservice-account-id
 - technically to return the session, you return it as a custom header with the response
+
+## Ignore directory
+
+This is a work in progress, but now if you don't want a folder to be parsed by apidoc (not going to be shown in the docs), the folder must start with:
+
+```
+_kintoignore
```

Comment:
Show examples of actual kintoignore? (multiple files with comments on which one are ignored and which one arent)